services/response_code.proto

```diff
@@ -1397,4 +1397,19 @@ enum ResponseCodeEnum {
    * Admin key is not set on token
    */
   TOKEN_HAS_NO_ADMIN_KEY = 337;
+
+  /**
+   * The node has been marked as deleted
+   */
+  NODE_DELETED = 338;
+
+  /**
+   * A node is not found
+   */
+  INVALID_NODE_ID = 339;
+
+  /**
+  * gossip_endpoint has a fully qualified domain name instead of ip
+  */
+  INVALID_GOSSIP_ENDPOINT = 340;
```

Comment:
Seems like there might be a lot more response codes based upon the error scenarios present in each transaction's comments.

Reply:
Still think we should identify those now during the design phase and reduce churn on protobuf repo.

Reply:
Added more response codes.